dhcpv4/dhcpv4.go

```diff
@@ -557,8 +572,15 @@ func (d *DHCPv4) SetOptions(options []Option) {
 }
 
 // AddOption appends an option to the existing ones.
+// If there is an OptionEnd in the list, it will insert it before that option.
```

Comment:
more precisely - if the last option is an OptionEnd, it will be inserted before that. A malformed packet may contain more than one OptionEnd, or more in general it may not be the last. We should either update this comment, or scan for the first OptionEnd and insert the new option there

Reply:
Ok, I will update the comment.